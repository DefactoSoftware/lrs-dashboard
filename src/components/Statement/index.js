import React from 'react';

export default ({ statement })=> {
  console.log(statement);
  return (
    <p>
      <span>{statement.actor}</span>{' '}
      <strong>{statement.verb}</strong>{' '}
      <span>{statement.object}</span>
    </p>
  );
}
