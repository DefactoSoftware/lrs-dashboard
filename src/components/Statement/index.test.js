import React from 'react';
import Statement from './index.js';
import renderer from 'react-test-renderer';

it('renders no statements when no statements are present', () => {
  const statement = {
    id: "1234",
    actor: 'foo@bar.com',
    verb: 'created',
    object: 'Chapter',
  };

  const tree = renderer.create(<Statement statement={statement} />).toJSON();

  expect(tree).toMatchSnapshot();
});
