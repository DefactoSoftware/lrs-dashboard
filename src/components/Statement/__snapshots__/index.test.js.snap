exports[`test renders no statements when no statements are present 1`] = `
<p>
  <span>
    foo@bar.com
  </span>
   
  <strong>
    created
  </strong>
   
  <span>
    Chapter
  </span>
</p>
`;
